src/main/java/org/apache/sysml/udf/lib/GenericFunction.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.sysml.udf.lib;
+
+import java.io.IOException;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.sysml.api.ExternalUDFRegistration;
+import org.apache.sysml.runtime.DMLRuntimeException;
+import org.apache.sysml.udf.FunctionParameter;
+import org.apache.sysml.udf.Matrix;
+import org.apache.sysml.udf.PackageFunction;
+import org.apache.sysml.udf.Scalar;
+
+import scala.Function0;
+
+public class GenericFunction extends PackageFunction {
+	private static final long serialVersionUID = -195996547505886575L;
+	String [] fnSignature;
+	FunctionParameter [] returnVals;
+	Function0<FunctionParameter []> scalaUDF;
+	public String _functionName;
+	public String _namespace;
+	
+	public void initialize() {
+		if(_namespace != null && !_namespace.equals(".defaultNS")) {
```

Comment:
@niketanpansare Could you substitute DMLProgram.DEFAULT_NAMESPACE for the String constant here?